floodlight/io/catapultApi.py

```diff
@@ -256,28 +271,40 @@ def get_meta_data(response_data_json_list: List[Dict]) -> Tuple[Dict[str, Dict[s
         "number": "jersey",
     }
 
-    for data_i in response_data_json_list:
-        for entry in data_i["data"]:
-            t.append(int(entry["ts"]) * 100 + int(entry["cs"]))
-
+    from collections import Counter
+    max_count = 0
+    for athlete_entry in response_data_dict_list:
+        athlete_t = []
+        for entry in athlete_entry["data"]:
+            t.append(int(entry["ts"]) * 100 + int(entry["cs"])) # timestamps are in centiseconds. Magic number 100 is needed for conversion to

```

Comment:
I think this is missing something at the end

Reply:
done